public/javascripts/group.js

```diff
@@ -57,9 +57,10 @@ const postForm = async (url, formData) => {
 
 addGroupForm.addEventListener("submit", async (e) => {
   const formData = new FormData(addGroupForm);
-  e.preventDefault();
+  // e.preventDefault();
```

Comment:
If this is not in use, let's remove this line of code.

Reply:
Removed